content/agent/basic_agent_usage/_index.md

```diff
@@ -1,4 +1,95 @@
 ---
-title: Basic Agent Usage
-external_redirect: /agent/basic_agent_usage/amazonlinux/
+title: Basic Agents Usage
+kind: documentation
 ---
+
+{{< partial name="platforms/platforms.html" >}}
+
+## Configuration management tools
+
+Manage the Datadog Agent and [integrations](/integrations) using configuration management tools:
+
+### Chef
+* [Chef Github project](https://github.com/DataDog/chef-datadog)
+* [Chef installation](https://app.datadoghq.com/account/settings#integrations/chef)
+* [Chef documentation](/integrations/chef)
+
+### Puppet
+* [Puppet Github project](https://github.com/DataDog/puppet-datadog-agent)
+* [Puppet installation](https://app.datadoghq.com/account/settings#integrations/puppet)
+* [Puppet documentation](/integrations/puppet)
+
+### Ansible
+* [Ansible Github project](https://github.com/DataDog/ansible-datadog)
+* [Ansible installation](https://app.datadoghq.com/account/settings#agent/ansible)
+* [Ansible documentation](/integrations/ansible/)
+
+Chef, Puppet, and Ansible [integrations](/integrations) use our public APIs, so if you're interested in using another automation tool, the above could be leveraged as examples to get you started.
```

Comment:
is it correct to point to integrations here? It might be confusing since we _do_ have an integrations for -let's say- Chef, but the context here is to use Chef as a tool to install the agent

Reply:
👌